apps/gdal_utils_priv.h

```diff
@@ -243,6 +243,8 @@ std::string CPL_DLL GDALGridGetParserUsage();
 
 std::string CPL_DLL GDALBuildVRTGetParserUsage();
 
+std::string CPL_DLL GDALContourGetParserUsage();
```

Comment:
as gdal_contour is a bin only utility, this function should not be exposed in gdal_utils_priv.h as it can't be used by the GDAL library

Reply:
done